src/glfw-sandbox/main.cpp

```diff
@@ -171,7 +67,7 @@ int main() {
 
         // Rendering will go here
         
-        triangleZ -= 0.001f;
+        triangleZ -= 0.001f; //What is the difference between 0.001f and 0.001? Don't know how to google this question :( 

```

Comment:
If C++ does not require us to write f, we maybe don't need to. It simply indicates the number should be treated as float.

Reply:
I've been reading some stuff on StackOverflow but they are really old. Maybe we should ask Hana?

Reply:
I know the answer! The result might surprise you :D Stay tuned for the afternoon session :)